adcircpy/forcing/tides/hamtide.py

```diff
@@ -20,11 +20,14 @@ def __init__(self, hamtide_dataset_directory: PathLike = None):
         if hamtide_dataset_directory is None:
             hamtide_dataset_directory = self.OPENDAP_URL
 
-        if Path(hamtide_dataset_directory).exists():
-            hamtide_dataset_directory = Path(hamtide_dataset_directory)
-            if len(list(hamtide_dataset_directory.glob('*.nc'))) == 0:
-                raise FileNotFoundError(f'no NetCDF files found at '
-                                        f'"{hamtide_dataset_directory}"')
+        try:
+            if Path(hamtide_dataset_directory).exists():
+                hamtide_dataset_directory = Path(hamtide_dataset_directory)
+                if len(list(hamtide_dataset_directory.glob('*.nc'))) == 0:
+                    raise FileNotFoundError(f'no NetCDF files found at '
+                                            f'"{hamtide_dataset_directory}"')
+        except OSError:
+            pass
```

Comment:
this constructor would break if a URL was given as the resource string

Reply:
This is a variable that you introduced, my system was different.

Reply:
> this constructor would break if a URL was given as the resource string

Users will never give URL as resource, only system paths.

Reply:
ok, that makes sense, I can revert this then

Reply:
reverting